sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -74,6 +80,19 @@ object SQLMetrics {
   private val TIMING_METRIC = "timing"
   private val AVERAGE_METRIC = "average"
 
+  private val baseForAvgMetric: Int = 10
```

Comment:
I'm not quite sure this hack worth. For small number of probes, we don't care the values I think. For large number of probes, having one more digit in the fraction part is not very useful.